tests/pytests/test_config.py

```diff
@@ -859,6 +859,74 @@ def _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
         _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
                                    testValue)
 
+
+@skip(cluster=False, redis_less_than='8.1.0')
+def testConfigAPIRunTimeOssGlobalPassword():
+    env = Env(noDefaultModuleArgs=True)
+    if env.env != 'oss-cluster':
+        env.skip()
+
+    env.expect('CONFIG', 'GET', 'search-oss-global-password')\
+        .equal(['search-oss-global-password', 'Password: *******'])
+
+    env.expect('CONFIG', 'SET', 'search-oss-global-password', '123')\
+        .error().contains('CONFIG SET failed')
+
+
+@skip(cluster=False, redis_less_than='8.1.0')
+def testConfigAPIRunTimeOssACLUser():
+    env = Env(noDefaultModuleArgs=True)
+    if env.env != 'oss-cluster':
+        env.skip()
+
+    env.expect('CONFIG', 'GET', 'search-oss-acl-username')\
+        .equal(['search-oss-acl-username', 'default'])
+
+    env.expect('CONFIG', 'SET', 'search-oss-acl-username', 'myUser')\
+        .error().contains('CONFIG SET failed')
+
+
+@skip(cluster=False, redis_less_than='8.1.0')
+def testClusterConfigFileOssGlobalPassword():
+    # Test using only redis config file
+    redisConfigFile = '/tmp/testClusterConfigFileOssGlobalPassword.conf'
+
+    # create redis.conf file in /tmp
+    if os.path.isfile(redisConfigFile):
+        os.unlink(redisConfigFile)
+    with open(redisConfigFile, 'w') as f:
+        f.write('search-oss-global-password mySecretPassword\n')
+
+    # Start the server using the conf file
+    env = Env(noDefaultModuleArgs=True, redisConfigFile=redisConfigFile)
+    if env.env != 'oss-cluster':
+        env.skip()
+
+    env.expect('CONFIG', 'GET', 'search-oss-global-password')\
```

Comment:
Can use a `CONFIG SET` command and verify it is written to the file (after `CONFIG REWRITE`)

Reply:
We can't use `CONFIG SET` because it is immutable.
I ran a local tests removing the immutable flag for `search-oss-global-password` and `search-oss-acl-username`,  the test updates their values and calls `CONFIG REWRITE`, the values saved to the file are the values restored by the getter functions.
```sh
# values saved to redis.conf file
search-oss-acl-username "modified_user_name"
search-oss-global-password "Password: *******"
```

Reply:
Right. If we cannot read it (since it is redacted which is good), can we use it with `auth`?
```
AUTH default mySecretPassword
```